src/coreclr/jit/lower.cpp

```diff
@@ -1091,6 +1091,15 @@ GenTree* Lowering::NewPutArg(GenTreeCall* call, GenTree* arg, fgArgTabEntry* inf
             MakeSrcContained(arg, arg->AsObj()->Addr());
         }
     }
+#elif defined(TARGET_LOONGARCH64)
+    if (type == TYP_STRUCT)
```

Comment:
This looks exactly like the `TARGET_ARMARCH` block. Combine then under `#if defined(TARGET_ARMARCH) || defined(TARGET_LOONGARCH64)`?